src/uk/gov/hmcts/contino/TeamNames.groovy

```diff
@@ -37,7 +37,7 @@ class TeamNames {
                       'plum-shared-infrastructure' : 'CNP',
                       'sscs':'SSCS',
                       'sscs-cor':'SSCS',
-                      'sscs-tya':'SSCS',
+                      'sscs-track-your-appeal-frontend':'SSCS',
```

Comment:
that's not a product name, that's an application name.
Changing a product name will cause the dns name to change for an application, it's not something we normally do, why is it being changed?

normally a product would be something like:
`sscs`

if a complete separate team is running it then it might be:
`sscs-tya`
`tya`

`track-your-appeal`

`sscs-track-your-appeal`

but including frontend makes it a `component`